EPRN.Common/Constants/Constants.cs

```diff
@@ -69,6 +69,8 @@ public static class PRNS
                     public const string Cancelled = "Cancelled";
                     public const string RequestCancelConfirmed = "RequestCancelConfirmed";
                     public const string DecemberWaste = "DecemberWaste";
+                    public const string DestinationPrn = "DestinationPrn";
+                    public const string PrnSavedAsDraftConfirmation = "PrnSavedAsDraftConfirmation";
```

Comment:
I would have a snappier action name if it were me: DraftConfirmation. I don't think there's any need for it to be any longer than that. It's already in a controller called PRN, so we don;t need that. The rest is implied.

Reply:
Done.